pkg/proxy/ipvs/proxier_test.go

```diff
@@ -4811,7 +4811,7 @@ func TestTestInternalTrafficPolicyE2E(t *testing.T) {
 			realServers1, rsErr1 := ipvs.GetRealServers(virtualServers1[0])
 			assert.NoError(t, rsErr1, "Expected no error getting real servers")
 
-			assert.Len(t, realServers1, tc.expectLocalRealServerNum, fmt.Sprintf("Expected %d real servers", tc.expectLocalRealServerNum))
+			assert.Lenf(t, realServers1, tc.expectLocalRealServerNum, "Expected %d real servers", tc.expectLocalRealServerNum)
```

Comment:
I think this has already been covered by #127649

https://github.com/kubernetes/kubernetes/pull/127649/files#diff-3dbe81044ed169e4ba160845c6ec6833993e4229fc17bec29770aac5f67ff317R4814

Reply:
yup. thanks for the PR but someone else beat you to it.